domains/ahnaf.json

```
@@ -0,0 +1,10 @@
+{
+  "description": "For my personal portfolio website",
+  "owner": {
+    "username": "AhnafFarhanHossain",
+    "email": "ahnaffarhanhossain@gmail.com"
+  },
+  "record": {
+    "A": ["103.62.149.9"]
+  }
```

you gave a vercel website for preview, this isnt vercel's ip
```suggestion
    "A": ["103.62.149.9"]
  }
```

---

I will manually host my portfolio on my server. That was my server's IP address. Vercel one is only for preview purposes; I will host the website on my server.

If you'd like to preview it on the same IP address, here it is: 

https://ahnaf.tawhid-dev.icu/